api/v1beta1/conversion.go

```diff
@@ -1,14 +1,26 @@
 package v1beta1
 
 import (
-	"github.com/vmware/cluster-api-provider-cloud-director/api/v1beta2"
+	"github.com/vmware/cluster-api-provider-cloud-director/api/v1beta3"
 	"k8s.io/apimachinery/pkg/conversion"
 )
 
-func Convert_v1beta2_VCDMachineTemplateResource_To_v1beta1_VCDMachineTemplateResource(in *v1beta2.VCDMachineTemplateResource, out *VCDMachineTemplateResource, s conversion.Scope) error {
-	return autoConvert_v1beta2_VCDMachineTemplateResource_To_v1beta1_VCDMachineTemplateResource(in, out, s)
+func Convert_v1beta3_VCDMachineTemplateResource_To_v1beta1_VCDMachineTemplateResource(in *v1beta3.VCDMachineTemplateResource, out *VCDMachineTemplateResource, s conversion.Scope) error {
+	return autoConvert_v1beta3_VCDMachineTemplateResource_To_v1beta1_VCDMachineTemplateResource(in, out, s)
 }
 
-func Convert_v1beta2_VCDMachineSpec_To_v1beta1_VCDMachineSpec(in *v1beta2.VCDMachineSpec, out *VCDMachineSpec, s conversion.Scope) error {
-	return autoConvert_v1beta2_VCDMachineSpec_To_v1beta1_VCDMachineSpec(in, out, s)
+func Convert_v1beta3_VCDMachineSpec_To_v1beta1_VCDMachineSpec(in *v1beta3.VCDMachineSpec, out *VCDMachineSpec, s conversion.Scope) error {
+	return autoConvert_v1beta3_VCDMachineSpec_To_v1beta1_VCDMachineSpec(in, out, s)
+}
+
+func Convert_v1beta3_VCDClusterSpec_To_v1beta1_VCDClusterSpec(in *v1beta3.VCDClusterSpec, out *VCDClusterSpec, s conversion.Scope) error {
```

Comment:
I am wondering why these functions were in zz_generated_conversions.go for v1beta2

Reply:
I am not certain of this. There are some functions in the regular zz_generated_conversions, and some are missing.